### ISSUE TITLE
some rooms were missing from the map (2024)

### ISSUE THREAD

**Issue by digitalica** [loomchild/sojourner-web#145]
could very wel be problem in data, not in this app.

it was a bit of a puzzle to find UB5.132 for example

(not shown on map)

**Comment by loomchild** [comment]
No, it's the app. I didn't update the map. I am thinking to remove the building maps, since specific they are not too easy to update, (I always struggle with obtaining the images in advance). Besides, with more complex buildings, like U, multi-level K, I find them challenging to read and prefer to rely on physical signs. Do you think the building maps are useful?

**Comment by digitalica** [comment]
Thanks for the clarification.

To me the building maps are useful, even though i've visted fosdem before, so know where the buildings are, i like to be able to see where in the building the room is, to choose exact route. Also: distance is a factor. From K4, UA can be a long way. So please keep up the struggle! (and even if you fail, whats there now is better then nothing!)

However, all maps i've seen on fosdem are 'just' bitmap images (i even had to search where the number was on a map by scrolling around ;-) ) . ideally they it should be open street map,  or, if that doesn't support multi-level indoor navigation enough, some other format (quick search found OpenLevelUp). But that could be a lot of work to build / integrate / maintain...

In general: I have not seen a good solution for navigation on (business) conferences or trade fair (I remember CeBIT (Centrum der Büro- und Informationstechnik) in Hannover and much more recent EBACE in genava) yet. (We just want small scale google maps I think, preferably with accurate indoor positioning, and streetview) I'll stop dreaming now ;-)

**Comment by loomchild** [comment]
Thanks. I think I'll keep them simple for now, but make sure to update them before each edition.

**Comment by loomchild** [comment]
BTW, there's https://nav.fosdem.org, but I haven't really used it and I'm not sure how to integrate it - perhaps I should check it out.

**Comment by loomchild** [comment]
I will try to integrate https://nav.fosdem.org if I have a chance before Saturday.
